cmd/operator-verify/manifests/cmd.go

```diff
@@ -11,13 +11,17 @@ import (
 
 func NewCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
-		Use:   "manifests",
-		Short: "Validates all manifests in a directory",
-		Long: `'operator-verify manifests' validates a bundle in the supplied directory
-and prints errors and warnings corresponding to each manifest found to be
-invalid. Manifests are only validated if a validator for that manifest
-type/kind, ex. CustomResourceDefinition, is implemented in the Operator
-validation library.`,
+		Use:   "manifests <manifestDir>",
+		Short: "Validate all manifests in a directory",
+		Args:  cobra.ExactArgs(1),
```

Comment:
As noted in the commit/PR description, we're blindly accessing the `args[0]` array element within validating that element even exists:

```go
...
func manifestsFunc(cmd *cobra.Command, args []string) {
	bundle, err := manifests.GetBundleFromDir(args[0])
	if err != nil {
		log.Fatalf("Error generating bundle from directory: %s", err.Error())
	}
```

 We can ensure that a single argument has been provided at the cobra.Command level.